gamemodes/terrortown/gamemode/client/cl_lang.lua

```diff
@@ -167,12 +167,29 @@ function LANG.GetUnsafeLanguageTable()
 end
 
 ---
--- Returns a translated @{string} text (if possible) directly from the language table
--- @param string name string key identifier for the translated @{string}
--- @return nil|string
+-- Returns the reference to a language table if it exists
+-- @param string name string key identifier for the language
+-- @return nil|table
+-- @realm client
+function LANG.GetUnsafeNamed(lang_name)
+	lang_name = lang_name and string.lower(lang_name)
```

Comment:
Or?

Reply:
same as above

Reply:
Same as above

Reply:
The `or nil` we already talked about is missing